NinchatSDKSwift/Implementations/Helpers/NINLowLevelClientSession+Extension.swift

```diff
@@ -5,13 +5,16 @@
 //
 
 import Foundation
-import NinchatSDK
+import NinchatLowLevelClient
 
 extension NINLowLevelClientSession {
-    func send(_ param: NINLowLevelClientProps, _ payload: NINLowLevelClientPayload? = nil) throws -> Int {
-        var actionID: Int64 = 0
-        try self.send(param, payload: payload, actionId: &actionID)
```

Comment:
Oh, the actionId is an *out* parameter, not an *in* parameter. (I thought this send function was something which stuffs the action id into the params object.)

I think the original code should be preserved, it looks less complicated. :)

Reply:
I can take it back, no worries :)

Reply:
I reverted this function to the previous commit.